Rn okta364354 remove dev console (#1905)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/setup-first-mobile-app/android/addredirectscheme.md

    ]
```
### Native sign in
To compile your project, you need to set up the `AuthenticationClient` with an `Org URL`.

> The `Org URL` can found on the Admin Console's global header in the upper-right corner. Click the section which displays your email and company name.  A drop-down menu will appear and display general org information including the `Org URL` (e.g. subdomain.okta.com).

Reviewer: ```suggestion
You can find the `Org URL` in the Admin Console's global header in the upper-right corner of the page. Click the section that displays your email and company name.  A drop-down box appears and displays general org information including the `Org URL` (for example, subdomain.okta.com).
```

> Once you find your `Org URL`, add the following to your `local.properties` file:

Reviewer: ```suggestion
After you find your `Org URL`, add the following to your `local.properties` file:
```
